main/src/ca/uwaterloo/flix/language/phase/Inliner.scala

```diff
@@ -18,21 +18,289 @@ package ca.uwaterloo.flix.language.phase
 
 import ca.uwaterloo.flix.api.Flix
 import ca.uwaterloo.flix.language.CompilationMessage
```

Comment:
Add your names at the top of the files. This is your work and you should get credit :)